spec/features/user_sends_message_to_client_spec.rb

```diff
@@ -48,4 +51,54 @@
       expect(page).to have_css "tr##{dom_id(savedsecondclient)} td", text: '7 days'
     end
   end
+
+  skip 'user schedules a message to client', :js do
```

Comment:
It'll fail on master until we've finished implementing the rest of the UI changes as this test is based on the final UI designs.